releasenotes/notes/cncf-ebpf-enable.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+issue:
+  - 47257
+releaseNotes:
+  - |
+    **Enabled** eBPF support for ambient is re-enabled after CNCF established guidance around dual-licensed eBPF bytecode
```

Comment:
I don't think we've ever mentioned anything about eBPF support to users in previous releases, so this might be considered a new feature rather than a bug fix or re-enablement.

Reply:
Thanks for pointing out this. Updated it.

Reply:
https://istio.io/latest/blog/2023/ambient-ebpf-redirection/